lib/features/mailbox_dashboard/presentation/mailbox_dashboard_bindings.dart

```diff
@@ -7,7 +7,7 @@ class MailboxDashBoardBindings extends Bindings {
 
   @override
   void dependencies() {
-    Get.lazyPut(() => MailboxDashBoardController());
+    Get.put(MailboxDashBoardController());
```

Comment:
why?

Reply:
We need it to initialize with the same view. Instead of being called when find.